### PR TITLE
Blog post: A new kind of Open Source Event

### LINKED ISSUE #78: Blog post: a new kind of open source event
Hello Editorial Team 👋

I will co-host a "Welcome to Open Source" event / workshop in Atlanta in April, and I wrote a blog post to introduce the idea behind it to the world, and also to send it out to other Open Source projects so that they can participate. I’ve started a Pull Request to our blog here:
hoodiehq/hood.ie#253

### PR DESCRIPTION
**Pull request by gr2m**
Based on draft at https://docs.google.com/document/d/1s4x2XsWjU729luQCwkJ2Dhne4PM8gLPXZMaHkBD9L8Y/edit?pref=2&pli=1

Preview: https://github.com/hoodiehq/hood.ie/blob/blog-post-a-new-kind-of-open-source-event/_posts/2016-03-15-a-new-kind-of-open-source-event.md

---

closes https://github.com/hoodiehq/editorial/issues/78

### CODE REVIEW

File: _posts/2016-03-15-a-new-kind-of-open-source-event.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+# After the event
+
+All prepared issues that have not been worked on during the event will be free for everyone to take. We hope that the participating projects will find joy in preparing starter issues, even without an event happening, to be able to welcome new contributors to Open Source at all times.
+
+If you would like to host a “Welcome to Open Source” event in your city, [let us know](http://hood.ie/contact/)! We are more than happy to assist.
+
+Thanks to Alex, Jan & Jenn for their reviews!
+
+The [online Word editor](http://wordhtml.com/ "Online Word Editor") will help you compose easily documents for the web.
```
Comment (jennwrites) [inline _posts/2016-03-15-a-new-kind-of-open-source-event.md]: What's this link?

**Comment by jennwrites** [comment]
Great job on this, I had no idea we were doing these events! Exciting stuff :smile: +1 to publish

File: _posts/2016-03-15-a-new-kind-of-open-source-event.md (hunk 1 above)
Comment (gr2m) [inline _posts/2016-03-15-a-new-kind-of-open-source-event.md]: LOL I used this editor in the process from Google Doc -> Markdow. What a scam to their ad in there, I totally missed it. Thanks for catching it :)

**Comment by gr2m** [comment]
> I had no idea we were doing these events

It kinda randomly happend :) Thanks for the review, I’ve removed the weird link and fixed up the commits. How is the process now, can we publish this now? Do we need another review? Is there a schedule when this post should go live?

**Comment by jennwrites** [comment]
I think it best that each post gets reviewed by 2 people, as a rule.

As far as a schedule goes, if you want to get it out tonight, I’m okay with pushing this out there. Otherwise if it can wait until tomorrow, (which gives more time for review) we can do that too. 

-- 
Jenn Turner
I ❤️ great email at jennwrit.es/newsletters 

On March 15, 2016 at 3:51:28 PM, Gregor Martynus (notifications@github.com) wrote:

I had no idea we were doing these events

It kinda randomly happend :) Thanks for the review, I’ve removed the weird link and fixed up the commits. How is the process now, can we publish this now? Do we need another review? Is there a schedule when this post should go live?

**Comment by gr2m** [comment]
okay thanks :) Let’s wait for a second review

**Comment by HipsterBrown** [comment]
Very awesome plan and event! I really like the reserved starter issues and openness. I'm looking forward to the results. :+1: LGTM!

**Comment by gr2m** [comment]
@renrutnnej we have this integration with https://lgtm.co/, which requires two people from the maintainers team to comment with LGTM before it allows us to merge something. I did this just now so we have two, just wanted to let you know for the future :)

**Pull request merged**